libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/distanceinfo/GantryDistanceInfo.kt

```diff
@@ -0,0 +1,56 @@
+package com.mapbox.navigation.base.trip.model.roadobject.distanceinfo
+
+import com.mapbox.navigation.base.trip.model.roadobject.RoadObjectType
+
+/**
+ * GantryDistanceInfo contains information about distance to the road object represented as gantry.
+ *
+ * @param distance distance to the gantry object
+ */
+class GantryDistanceInfo internal constructor(
+    roadObjectId: String,
+    @RoadObjectType.Type roadObjectType: Int,
+    val distance: Double,
```

Comment:
Why do we need to expose this field if it's equal to `distanceToStart`? Let's hide it.

Reply:
done